packages/components/src/call-out/test/index.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * @jest-environment jsdom
+ */
+import { fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import '@testing-library/user-event';
+import { Button, __experimentalText as Text } from '@wordpress/components';
+import React from 'react';
+import { CallOut } from '../index';
+
+describe( 'CallOut', () => {
+	test( 'renders title and description', () => {
+		render(
+			<CallOut
+				title="Test Title"
+				description={
+					<Text as="p" variant="muted">
+						Helpful content
+					</Text>
+				}
+			/>
+		);
+		expect(
+			screen.getByRole( 'heading', {
+				name: 'Test Title',
+			} )
+		).toBeInTheDocument();
+		expect( screen.getByRole( 'paragraph' ) ).toHaveTextContent( 'Helpful content' );
+	} );
+
+	test( 'variant prop applies correct class', () => {
+		const { container, rerender } = render(
+			<CallOut
+				title="Test Title"
+				description={
+					<Text as="p" variant="muted">
+						Helpful content
+					</Text>
+				}
+				variant="highlight"
+			/>
+		);
+		expect( container.firstChild ).toHaveClass( 'call-out--highlight' );
+
+		rerender(
+			<CallOut
+				title="Test Title"
+				description={
+					<Text as="p" variant="muted">
+						Helpful content
+					</Text>
+				}
+				variant="default"
+			/>
+		);
+		expect( container.firstChild ).toHaveClass( 'call-out--default' );
+	} );
+
+	test( 'action buttons are clickable', () => {
```

Comment:
It seems that this is testing `Button` and not `Callout`. At most, I would only test that what is passed into `actions` is `.toBeVisible`.

Relatedly, it can be a maintenance burden to use unnecessary wp-components (`Text`, `Button`) in these tests, as the tests may need to be updated when the upstream component changes. Plain HTML elements are preferable when the functionality is unrelated to the actual component being tested.